platform/ios/src/MGLAnnotationView.mm

```diff
@@ -1,10 +1,12 @@
 #import "MGLAnnotationView.h"
 #import "MGLAnnotationView_Private.h"
 #import "MGLMapView_Private.h"
+#import "MGLCalloutView.h"
```

Comment:
To answer my own question: Yes, because of the fixing of `MGLMapView_Private.h`. 

Because of this we need to add

```objc
#import "MGLPointAnnotation.h"
```

to silence a warning below in

https://github.com/mapbox/mapbox-gl-native/blob/d8e5c5afcb8c0184012380e651881db457792af7/platform/ios/src/MGLAnnotationView.mm#L322